src/main/java/com/java_essential/Cherednichenko/extra_task/CustomerService/Service.java

```diff
@@ -0,0 +1,51 @@
+package com.java_essential.Cherednichenko.extra_task.CustomerService;
+
+import java.util.List;
+
+public class Service {
+    private int id;
+    private String serviceName;
+    private String isActive;
+    private List<Integer> _listCustomer;
```

Comment:
```suggestion
    private List<Integer> _listCustomer;
```
incorrect naming `_listCustomer` change it according to java code convention rules
[https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html](url)